.circleci/config.yml

```diff
@@ -206,6 +207,18 @@ jobs:
             --build-arg APP_GIT_COMMIT=${CIRCLE_SHA1}
             --build-arg APP_BRANCH_NAME=${CIRCLE_BRANCH}
 
+  e2e-test-branch:
+    executor: crime-forms-end-to-end-tests/e2e-test-executor
+    steps:
+      - crime-forms-end-to-end-tests/run-e2e-tests:
+          e2e_current_image_var: NSCC_APPSTORE_IMAGE
+          e2e_current_image: ${AWS_ECR_REGISTRY_ID}.dkr.ecr.${ECR_REGION}.amazonaws.com/${APPSTORE_ECR_REPOSITORY}:branch-${CIRCLE_SHA1}
+
```

Comment:
is it worth chucking an ` e2e_branch: main` here?

Reply:
yes, missed doing it in this repo 👍

Reply:
although, to be clear, main is the default in any event but our future selves may be grateful for the reminder of how to change the e2e branch